backend/src/service/generic.ts

```diff
@@ -82,8 +84,53 @@ export function genericService<T>(
     }
   };
 
+  /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
If you are using these to make the code easier to read, I think there are some vscode extensions that help you visually separate the functions when looking at them, :)
It's probably best to remove these before merging. I don't hgave anything against it personally, but it's not very standard hahaha

Reply:
Yeah. I agree with Mauricio.